### PR TITLE
Refactor persistent_cache_size to persistent_cache_size_mb

### PR DESCRIPTION
**Pull request by reidHoruff**
This diff refactors rocksdb_persistent_cache_size to rocksdb_persistent_cache_size_mb.

### CODE REVIEW

**Review by alxyang — APPROVED** [review]
add squash with message for rebasing later on

File: storage/rocksdb/ha_rocksdb.cc [hunk 1]
```diff
@@ -662,10 +662,10 @@ static MYSQL_SYSVAR_STR(
     nullptr, "");
 
 static MYSQL_SYSVAR_ULONG(
-    persistent_cache_size, rocksdb_persistent_cache_size,
+    persistent_cache_size_mb, rocksdb_persistent_cache_size_mb,
     PLUGIN_VAR_RQCMDARG | PLUGIN_VAR_READONLY,
     "Size of cache for BlockBasedTableOptions::persistent_cache for RocksDB",
```
Comment (vasilep) [inline storage/rocksdb/ha_rocksdb.cc]: "... in MB"

File: storage/rocksdb/ha_rocksdb.cc [hunk 2]
```diff
@@ -3394,14 +3394,15 @@ static int rocksdb_init_func(void *const p) {
     mysql_mutex_unlock(&rdb_sysvars_mutex);
   }
 
-  if (rocksdb_persistent_cache_size > 0) {
+  if (rocksdb_persistent_cache_size_mb > 0) {
     std::shared_ptr<rocksdb::PersistentCache> pcache;
+    uint64_t cache_size_bytes = rocksdb_persistent_cache_size_mb * 1024 * 1024;
```
Comment (vasilep) [inline storage/rocksdb/ha_rocksdb.cc]: style nit: no space before =

File: storage/rocksdb/ha_rocksdb.cc (hunk 1 above)
Comment (vasilep) [inline storage/rocksdb/ha_rocksdb.cc]: "... in MB"

Comment (vasilep) [inline storage/rocksdb/ha_rocksdb.cc]: "... in MB"

**Pull request closed without merging**